src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Person } from '../../types/Person';
+
+type Props = {
+  filterPerson: Person[];
+  handlerOnMouseDown: (person: Person) => void;
+  showPersonList: boolean;
+};
+
+const DropdownMenu: React.FC<Props> = React.memo(
+  ({ filterPerson, handlerOnMouseDown, showPersonList }) => {
+    return (
+      <>
+        {showPersonList && (
+          <div className="dropdown-menu" role="menu">
+            <div className="dropdown-content">
+              {filterPerson.length > 0 ? (
+                <>
```

Comment:
you can remove fragment in this case

Reply:
@StasSokolov 
without fragment I have `"Parsing error: ',' expected.eslint ',' expected.ts(1005) (property) filterPerson: Person[]"`

![image](https://github.com/mate-academy/react_autocomplete/assets/75209103/fe42690c-1083-4559-9a83-2980a3f84aaf)